module-2/backend/week_one.md

```diff
@@ -5,30 +5,100 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+ Get: retreive/display info, Post: display new info, Put:save new info, Delete: delete info, Patch: update/fix?
+
 2. What is Sinatra?
+
+ Sinatra is a web app framework.
+
 4. What is MVC?
+
+ MVC is models, views, controllers. It is a pattern put in place to organize information based on functionality.
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+ Because it's restful.
+ 
 6. What types of variables are accessible in our view templates without explicitly passing them?
+
+ params
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
+  @count = 1
   get '/horses' do
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+
+  ```ruby
+  @count = 1
+  params[name]="Mr.Ed"

```

Comment:
Generally don't want to set values in our params hash in our controller. Instead we use the values available to us that came in with the HTTP request.

Regarding an instance variable, we would make it available to the view as follows:

```ruby
get '/horses' do
  name = "Mr. Ed"
  erb :index, locals: {name: name}
end
```

Alternatively, without assigning "Mr. Ed" to a variable, we can do the following:

```ruby
get '/horses' do
  erb :index, locals: {name: "Mr. Ed"}
end
```